sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -70,6 +72,41 @@ internal class NextLinkOperationImplementation : IOperation
             return new OperationToOperationOfT<T>(operationSource, operation);
         }
 
+        public static IOperation Create(
+            HttpPipeline pipeline,
+            RehydrationToken? rehydrationToken,
+            string? apiVersionOverride = null)
+        {
+            Argument.AssertNotNull(rehydrationToken, nameof(rehydrationToken));
+
+            var lroDetails = ModelReaderWriter.Write(rehydrationToken!, new ModelReaderWriterOptions("J")).ToObjectFromJson<Dictionary<string, string>>();
+            if (!Uri.TryCreate(lroDetails["initialUri"], UriKind.Absolute, out var startRequestUri))
+                throw new InvalidOperationException("Invalid initial URI");
+            if (!lroDetails.TryGetValue("nextRequestUri", out var nextRequestUri))
+                throw new InvalidOperationException("Invalid next request URI");
```

Comment:
```suggestion
            if (!lroDetails.TryGetValue("nextRequestUri", out var nextRequestUri))
            {
                throw new InvalidOperationException("Invalid next request URI");
            }
```

There's way too much density here to skip braces and have a clear understanding of flow. 
 Please either add whitespace so that it is clear the `if` and `throw` lines are grouped or add explicit braces.   Personally, I'd prefer that you add the explicit braces.

Leaving one comment here, but this applies to all of the if/throw combinations in this block.

Reply:
updated